src/Orleans/Serialization/OrleansJsonSerializer.cs

```diff
@@ -1,27 +1,23 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net;
-using System.Runtime.Serialization.Formatters.Binary;
-using System.Text;
-using System.Threading.Tasks;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using Orleans.Runtime;
 
 namespace Orleans.Serialization
 {
-    internal class OrleansJsonSerializer : IExternalSerializer
+    public class OrleansJsonSerializer : IExternalSerializer
     {
-        public static JsonSerializerSettings settings;
+        public static JsonSerializerSettings SerializerSettings { get; set; }
```

Comment:
This settings contains converters for internal Orleans types, so making it's setter public means providing users great way of shooting in the leg. Users will be able to adjust settings they need or add their's custom converters as it have public getter.